crates/pallet-domains/src/lib.rs

```diff
@@ -423,10 +423,6 @@ mod pallet {
     #[pallet::storage]
     pub type SuccessfulBundles<T> = StorageMap<_, Identity, DomainId, Vec<H256>, ValueQuery>;
 
-    /// Fraud proofs submitted successfully in current block.
-    #[pallet::storage]
-    pub(super) type SuccessfulFraudProofs<T: Config> =
```

Comment:
why do we not need this anymore on the client? 
How is client going to fetch the what fraud proofs are included in the previous consensus block ?
What am I missing here ?

Reply:
This is previously required by the client to maintain all the mismatched ER in the aux storage, e.g. the client uses FP as a signal of bad ER pruning, but it is not more needed since https://github.com/subspace/subspace/pull/2329